guides/doc-Administering_Red_Hat_Satellite/topics/Security_Compliance_Management.adoc

```diff
@@ -5,6 +5,11 @@ Security compliance management is the ongoing process of defining security polic
 Any non-compliance is managed according to the organization's configuration management policies.
 Security policies range in scope from host-specific to industry-wide, therefore, flexibility in their definition is required.
 
+ifndef::satellite[]
+Note that the following chapter refers only to RPM-based systems.
+We welcome any and all equivalent documentation for Debian users.
```

Comment:
That can't work since we haven't packaged the plugins.

Reply:
We need to drop a note or hide this yeah. This won't work at the moment and unfortunately we need to hide.

This is a typical case when hiding whole chapters is not the best approach, see, a technically skilled user could see what Foreman offers here and it's not that difficult for someone with experience to deploy this on Debian even without packages available. This is why I prefer to drop a note and still show all the text.